feature-libs/product/configurators/rulebased/components/attribute/product-card/configurator-attribute-product-card.component.html

```diff
@@ -0,0 +1,92 @@
+<div
+  class="cx-configurator-attribute-product-card"
+  [ngClass]="{
+    'cx-configurator-attribute-product-card-selected': product.selected
+  }"
+>
+  <div class="cx-configurator-attribute-product-card-rows">
```

Comment:
think about rename all of the elements' classnames below, because we don't use BEM or anything. `cx-xxxxxx` classnames should probably go with some important elements, not containers, rows, imgs, etc

Reply:
changed